LSCS.Web/Scripts/Templates/Checklists.jsx

```diff
@@ -1 +1,44 @@
-﻿// Create JSX templates here
\ No newline at end of file
+﻿var Checklist = React.createClass({
+    render: function() {
+        return (
+            <tr>
+                <td>{this.props.data.Title}</td>
+                <td>{this.props.data.Description}</td>
+                <td>{this.props.data.FileNumber}</td>
+                <td>{this.props.data.Location}</td>
```

Comment:
Location is a composite object. Might need some wrangling to get a meaningful string value out of.